src/components/ListTemplate/ColumnTemplates/DefaultColumn/DefaultColumn.js

```diff
@@ -0,0 +1,49 @@
+import React, { memo } from 'react';
+import HighLight from 'react-highlighter';
+import PropTypes from 'prop-types';
+
+import { AppIcon } from '@folio/stripes/core';
+
+import sharedCss from '../../../../shared.css';
+
+export const DefaultColumn = memo(props => {
+  const {
+    value,
+    className,
+    iconKey,
+    searchTerm,
+  } = props;
+
+  const content = searchTerm ? (
```

Comment:
We have a code style rule about formatting ternary operators. Point 9 -https://wiki.folio.org/display/FOLIJET/Code+style+rules+not+covered+by+eslint